core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -635,6 +635,67 @@ public static function insertTagToSrc($data)
 		return $return;
 	}
 
+	/**
+	 * Adds {{env::path}} to relative links
+	 *
+	 * @param string $data The markup string
+	 *
+	 * @return string
+	 */
+	public static function addEnvPath($data)
+	{
+		$return = '';
+		$paths = preg_split('/((src|href)="([^"]+)")/i', $data, -1, PREG_SPLIT_DELIM_CAPTURE);
+
+		for ($i=0, $c=\count($paths); $i<$c; $i+=4)
+		{
+			$return .= $paths[$i];
+
+			if (!isset($paths[$i+1]))
+			{
+				continue;
+			}
+
+			if (preg_match('(^(https?://|#|/|\{\{))i', $paths[$i+3]))
```

Comment:
```suggestion
			if (preg_match('(^([0-9a-z+.-]+:|#|/|\{\{))i', $paths[$i+3]))
```

We have to keep any links with a protocol untouched I think (e.g. `tel:…`).